workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.ts

```diff
@@ -22,12 +21,12 @@ import { InvalidTokenError } from '../errors/invalidTokenError';
 import { InvalidTokenTypeError } from '../errors/invalidTokenTypeError';
 import { PluginConfigurationError } from '../errors/pluginConfigurationError';
 import { Tokens } from '../tokens';
-import { getTimeInSeconds } from '../utils';
+import { getTimeInSeconds, TimeUnits } from '../utils';
 
 interface TokensExpiration {
-  idToken?: number; // in seconds
-  accessToken?: number; // in seconds
-  refreshToken?: number; // in seconds
+  idToken: number; // in seconds
```

Comment:
more curiousity -- why seconds? is there a standard?

Reply:
changing to ms to more match the javascript standard